src/components/TodoContainer.js

```diff
@@ -62,11 +70,8 @@ const TodoContainer = ({ tableName }) => {
     }
   };
 
-  const handleChange = (id) => {
-    setIsChecked((prevState) => ({
-      ...prevState,
-      [id]: !prevState[id],
-    }));
+  const handleChange = async (id, done) => {
```

Comment:
ideally this would have a more specific name on all the components higher in the tree than the` checkbox` itself:

```suggestion
  const handleToggleTodoCheckbox = async (id, done) => {
```